### ISSUE TITLE
MySQL Support when running on linux using .NET Core distribution.

### ISSUE THREAD

**Issue by toby11** [ariacom/Seal-Report#51]
I have reporting working well on my dev machine using the mysql odbc drivers however how would that be done when running on .net core in linux connecting to a local mysql database?

or is that not possible?

**Comment by computamike** [comment]
That should be possible - I've not seen Seal-Report yet - but I have written ASP.NET sites, that used MySQL as a store.  Both as a local DB, and using MySQL Hosted on AWS.  

I might have a go at this myself - because this is a usecase that I am interested in.

**Comment by toby11** [comment]
yes its more complicated because seal reports requires an ODBC database driver for MySQL rather than using the native .net core Mysql.Data connector.

so the issue seems to be how to install odbc drivers in linux.

**Comment by zhoujunhe** [comment]
The latest version of Mysql.Data is already supported
